default/python/AI/AIDependencies.py

```diff
@@ -277,10 +277,12 @@ class Tags:
 PRO_SOL_ORB_GEN = "PRO_SOL_ORB_GEN"
 PRO_MICROGRAV_MAN = "PRO_MICROGRAV_MAN"
 PRO_SINGULAR_GEN = "PRO_SINGULAR_GEN"
-PROD_AUTO_NAME = "PRO_SENTIENT_AUTOMATION"
+PROD_AUTO_1_NAME = "PRO_ADAPTIVE_AUTOMATION"
+#PROD_AUTO_2_NAME = "PRO_SENTIENT_AUTOMATION"
```

Comment:
./AI/AIDependencies.py:281:1: E265 block comment should start with '# '

Reply:
Fixed